src/main/java/randoop/util/Randomness.java

```diff
@@ -90,6 +92,51 @@ public static int nextRandomInt(int i) {
     throw new BugInRandoopException();
   }
 
+  public static <T extends WeightedElement> T randomMemberWeighted(
```

Comment:
Every class, method, and field must have documentation giving its specification.  Please add it for the routines in this file.  I can't review code unless I know what it is supposed to do.
(I realize that a few parts of Randoop are missing the documentation that they need, but let's not contribute to that problem.)